juniper_junos-formula/tests/test_120_states.py

```diff
@@ -121,6 +121,7 @@ def test_apply(host, device, state, test):
             '+               interface-mode trunk;',
             '+               vlan {',
             '+                   members 1-2;',
+            '+   ge-0/0/5 {\n\+       disable;\n\+       mtu 1500;',
```

Comment:
There's an opening `{`, but no closing `}`. Is this intentional/correct?

Also, the other lines in diffs_shared are split at linebreaks instead of having `\n` embedded, which is easier to read. Can you please split the added line?

Reply:
I found the split lines actually make for broken tests, because re.search searches for the lines in any part of the output. But we need to check the settings are actually part of the relevant interface.

Reply:
Ah, the tests only search if each line can be found in the generated config? I didn't expect this (and I'd recommend to change it so that it checks against the whole expected config file - probably not in this MR), but at least it explains why you've done the addition in this way.

Since this was my only question on this PR, feel free to merge as is.

Reply:
Yes, I noticed this here and hence did not want to add to the same pattern.